### ISSUE TITLE
Issue with theme/token colors, css variables are not defined

### ISSUE THREAD

**Issue by kzlar** [tamagui/tamagui#2895]
### Current Behavior

Theme/Token colors with the same key/name appear to be "not defined" in CSS, breaking previously working websites

### Tamagui Version

```markdown
Broke in 1.109
```

### Platform (Web, iOS, Android)

```markdown
Web
```

### Reproduction

```markdown
https://codesandbox.io/p/devbox/rough-firefly-n9kltc?workspaceId=e575660c-ea91-4cf2-a255-d6014a7e15ee&layout=%257B%2522sidebarPanel%2522%253A%2522GIT%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clq4t6azo00063b6hajbkj1be%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clq4t6azo00023b6hthoisote%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clq4t6azo00033b6h28fo02rl%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clq4t6azo00053b6h0htynuqv%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clq4t6azo00023b6hthoisote%2522%253A%257B%2522id%2522%253A%2522clq4t6azo00023b6hthoisote%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0oc0aoe00023b6jdwd9u7m9%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252F.devcontainer%252Fdevcontainer.json%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522cm0oc1cvr00023b6ihsxj2w4x%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A13%252C%2522startColumn%2522%253A25%252C%2522endLineNumber%2522%253A13%252C%2522endColumn%2522%253A30%257D%255D%252C%2522filepath%2522%253A%2522%252Fapps%252Fnext%252Fpackage.json%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522activeTabId%2522%253A%2522cm0oc1cvr00023b6ihsxj2w4x%2522%257D%252C%2522clq4t6azo00053b6h0htynuqv%2522%253A%257B%2522id%2522%253A%2522clq4t6azo00053b6h0htynuqv%2522%252C%2522activeTabId%2522%253A%2522cm0oc3nd1000l3b6i1wynpaqs%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clq4t6azo00043b6h40mtr8e7%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%2522%257D%252C%257B%2522type%2522%253A%2522PROJECT_SETUP%2522%252C%2522id%2522%253A%2522cm0obzx7w002w3b6jcrbwyp5f%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%252C%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A3000%252C%2522id%2522%253A%2522cm0oc3nd1000l3b6i1wynpaqs%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%252C%2522clq4t6azo00033b6h28fo02rl%2522%253A%257B%2522id%2522%253A%2522clq4t6azo00033b6h28fo02rl%2522%252C%2522activeTabId%2522%253A%2522cm0oc0wir000z3b6jdqolur22%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522Development%2522%252C%2522id%2522%253A%2522cm0oc0wir000z3b6jdqolur22%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15.462962962962962%257D
```

**Comment by kzlar** [comment]
See https://discord.com/channels/909986013848412191/974145843919716412/1279459780376920190 for more info

**Comment by kzlar** [comment]
Fixed in 1.111.8.
Thanks Nate!
